linux-userns-guide.md

```diff
@@ -0,0 +1,53 @@
+[< Back to README](./README)
+
+Linux user namespaces guide
+---------------------------
+
+This guide details how to enable user namespaces on Linux. This is a required step for running the substra backend.
+
+On Linux systems, all the docker instances create files with `root` permissions.
+TO working correctly in a dev environment, we need the files created by our dockers to have the same rights as the ones we use to launch our celery tasks.
```

Comment:
Beware `TO working`

Reply:
Fixed https://github.com/SubstraFoundation/substra-backend/pull/39/commits/f22e2beb456b1576ddbe2fb630462c486c3124a9